### PR TITLE
Update Storage POMs to Use New Surefire Argline Pattern

### PR DESCRIPTION
**Pull request by alzimmermsft**
This PR updates the Storage POMs to use the new `maven-surefire-plugin` `argLine` configuration.

### CODE REVIEW

File: sdk/storage/azure-storage-file-datalake/pom.xml [hunk 1]
```diff
@@ -36,6 +36,9 @@
     <src.dir>src/main/java</src.dir>
     <test.dir>src/test/java</test.dir>
     <AZURE_TEST_SYSTEM_PLAYBACK_PROPERTIES_FILE>../azure-storage-common/ci.system.properties</AZURE_TEST_SYSTEM_PLAYBACK_PROPERTIES_FILE>
+
+    <!-- Configures the Surefire JVM to use a 6gb max heap. -->
+    <surefireXmx>4096m</surefireXmx>
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-datalake/pom.xml:41]: 6gb != 4096mb

I'd just drop these comments, the xml element seems to be clear enough.

**Review by kasobol-msft — APPROVED** [review]
LGTM. Please run nightly and possibly weekly pipe to validate.

**Comment by alzimmermsft** [comment]
> LGTM. Please run nightly and possibly weekly pipe to validate.

Nightly pipeline was ran, and passed, already (I ran it since the CI never ran)

**Pull request merged**